app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Posts page<h1>
```

Comment:
- [ ] You are missing the `show.html.erb` for both Post and Users. Based on the project requirement 👇🏾 , please add a show view file to both `/post/show.html.erb` and `/users/show.html.erb`.

![image](https://github.com/Chaseknis/blog_app/assets/84156714/c75e6463-7ef3-44d6-b90a-22c9cee1ec63)

Reply:
Thank you @bobb-Rob. Making the changes right now.